.travis/server.postgres.yaml

```diff
@@ -0,0 +1,113 @@
+# Pulp server configuration
+#
+# Values shown are the default values used, unless otherwise indicated.
+
+# Django Settings
+#
+# `allowed_hosts`: A list of strings representing the host/domain names that
+# Pulp can serve. This is a security measure to prevent HTTP Host header
+# attacks. A value beginning with a period can be used as a sub-domain wildcard.
+# The default is the host's FQDN.
+#
+# allowed_hosts:
+#   - pulp.example.com
+
+# `databases`: An associative array (dictionary) of databases to use. For the
+# full list of configuration options, refer to the Django database documentation
+# shipped with your version of Django, or online at
+# https://docs.djangoproject.com/en/1.8/ref/settings/#databases
+#
+# By default, Pulp will use a sqlite3 database. If your environment is used in production
+# consider configuring Postgresql or another Django supported database.
+#
+# databases:
+#   default:
+#     CONN_MAX_AGE: 0
+#     ENGINE: django.db.backends.sqlite3
+#     NAME: /var/lib/pulp/sqlite3.db
+#     USER:
+#     PASSWORD:
+#     HOST:
+#     PORT:
+
+databases:
+  default:
+    ENGINE: django.db.backends.postgresql_psycopg2
+    NAME: pulp
+    USER: pulp
+
+# `SECRET_KEY`: A secret key for a particular Django installation. This is used to provide
+# cryptographic signing, and should be set to a unique, unpredictable value.
+# Pulp does not provide a default secret key. This must be user provided in order for the
+# Pulp Django Application to run.
+# SECURITY WARNING: keep the secret key used in production secret!
+#
+# SECRET_KEY:
+
+SECRET_KEY: '%25(i14$828q74)@k3d7=rn2hjg1jd%^zwa92qc6nxsto2x@(#'
```

Comment:
This seemed like the quick and easy way to deal with SECRET_KEY and I'm not sure if it matters if we hardcode it in Travis?